src/content/cookbook/plugins/play-video.md

```diff
@@ -85,6 +85,25 @@ The `video_player` plugin can only play asset videos in iOS simulators.
 You must test network-hosted videos on physical iOS devices.
 :::
 
+### MacOS
+
+If you use network-based videos, [add the
+`com.apple.security.network.client`
+entitlement](https://docs.flutter.dev/platform-integration/macos/building#entitlements-and-the-app-sandbox)
+
+### Web
+
+> The Web platform does **not** support `dart:io`, so avoid using the `VideoPlayerController.file`
+> constructor for the plugin. Using the constructor attempts to create a
+> `VideoPlayerController.file` that will throw an `UnimplementedError`.
```

Comment:
Why are these lines prefixed with an angle bracket?

```suggestion
### web

Flutter web does **not** support `dart:io`,
so avoid using the `VideoPlayerController.file` constructor for the plugin.
Using this constructor attempts to create a`VideoPlayerController.file`
that throws an `UnimplementedError`.
```

Reply:
I just took the formatting from the other readme file. I don't know exactly which markdown flavor is used here